src/auth0/WorkersInterceptors.tsx

```diff
@@ -5,7 +5,7 @@ import { useJwtFromCookie } from '../Redux/hooks';
  
 
 const workerInstance = axios.create({
-  baseURL: import.meta.env.VITE_WORKERS_SERVER_URL,
+  baseURL: import.meta.env.WORKERS_SERVICE_URL,
```

Comment:
אני חושבת שאם אין לזה תחילית של vite_ זה לא עובד

Reply:
סדרנו את השמות של המשתני סביבה ועשינו שמות חדשים - נשלח את העדכונים של ה ENV
וכאן נחמי אמרה שפחות קשור VITE אז שיננו...

Reply:
זה עובד?
לא אכפת לי איך תקראי לזה
אני רק רוצה לוודא שבדקת וזה עובד גם אם אין לזה את התחילית

Reply:
את מתכוונת שהשם פה משמעותי ברמה של נפילת קוד ?
לא את כל הקריאות שרת יש לנו איך להגיע אליהם דרך ה react  - מה שבדקנו זה שקיים קריאת שרת כזאת עם אותו שם והתאמה ל base url וכו'
את ה inventory בדקנו וזה עבד מצוין
ספציפי בקוד בינתיים אין שימוש בכלל לא ב InfraInstance ולא ב workerInstance

Reply:
כן, אני מתכוונת שאם השם לא מתחיל ב vite הוא לא יקבל את הערך, והמשתנה יהיה undefined
https://vitejs.dev/guide/env-and-mode

Reply:
תוקן
זה באמת לא תקין לכתוב בלי ה VITE